CRM/Contact/Form/Task/PDFTrait.php

```diff
@@ -272,7 +272,7 @@ public function postProcess(): void {
 
     if ($tee) {
       $tee->stop();
-      $content = file_get_contents($tee->getFileName(), NULL, NULL, NULL, 5);
+      $content = file_get_contents($tee->getFileName(), FALSE, NULL, 0, 5);
       if (empty($content)) {
```

Comment:
This seems fine.
An alternate below since the contents aren't what it's trying to really get at, but ... meh ...

```suggestion
      if (empty(filesize($tee->getFileName()))) {
```

(also covers FALSE in case of some other error)